src/admin/title/index.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+export const Title = () => <Grid container justify={'center'}>
```

Comment:
Let's just have a simple `title.jsx` file, without the folder.

Reply:
done